src/common/functions.js

```diff
@@ -0,0 +1,117 @@
+export const s4 = () => {
+  return Math.floor((1 + Math.random()) * 0x10000)
+    .toString(16)
+    .substring(1);
+};
+
+// generates uniqueID
+export const uniqueId = () => {
+  return s4() + s4() + '-' + s4() + '-' + s4() + '-' + s4();
+};
+
+// check singular or plural time for "ago"
+export const pluralCheck = s => {
+  if (s == 1) {
+    return ' ago';
+  } else {
+    return 's ago';
+  }
+};
+
+// function to convert timestamp to readable time
+export const timeConverter = timestamp => {
```

Comment:
what is the format of timestamp that this takes in?

Can we write a precondition/throw errors if input is incorrect (ie integer, string, boolean, null, etc) 
^ To write these as tests.